### PR TITLE
[CHK-2792] feat(closepayment): implement `closePayment` for PayPal payments

### PR DESCRIPTION
**Pull request by giovanniberti**
Depends on https://github.com/pagopa/pagopa-infra/pull/1881

#### List of Changes

 * refactor `closePayment` handling code
 * implement `closePayment` for PayPal payments
 * implement `closePayment` for MyBank payments
 * implement `closePayment` for Bancomatpay payments

#### Motivation and Context

Implement request generation for `closePayment` API in case of APM payments (PayPal, Bancomatpay and MyBank).
After this PR the correct information will be propagated to the rest of the pagoPA platform when a (guest) PayPal payment is concluded.

#### How Has This Been Tested?

[TODO]

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

Depends on pagopa/pagopa-infra#1881

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -355,7 +355,7 @@
 						</goals>
 						<configuration>
 							<inputSpec>
-								https://raw.githubusercontent.com/pagopa/pagopa-infra/main/src/core/api/nodopagamenti_api/nodoPerPM/v2/_openapi.json.tpl
+								https://raw.githubusercontent.com/pagopa/pagopa-infra/CHK-2674-additionalpaymentinformation-paypal/src/core/api/nodopagamenti_api/nodoPerPM/v2/_openapi.json.tpl
```
Comment (pietro-tota) [inline pom.xml:358]:
```suggestion
								https://raw.githubusercontent.com/pagopa/pagopa-infra/main/src/core/api/nodopagamenti_api/nodoPerPM/v2/_openapi.json.tpl
```

TODO: to be replaced after dependent infra PR merge

**Review by pietro-tota — APPROVED** [review]
🚀

**Review by pietro-tota — APPROVED** [review]

File: pom.xml [hunk 2]
```diff
@@ -590,6 +590,8 @@
 					<connectionUrl>scm:git:https://github.com/pagopa/pagopa-ecommerce-commons.git</connectionUrl>
 					<scmVersionType>tag</scmVersionType>
 					<scmVersion>${pagopa-ecommerce-commons.version}</scmVersion>
+					<scmVersionType>branch</scmVersionType>
+					<scmVersion>CHK-2736-wallet-details</scmVersion>
```
Comment (pietro-tota) [inline pom.xml:594]:
```suggestion
```
TODO: to be removed after dependend commons pr merge

**Review by pietro-tota — APPROVED** [review]

**Review by pietro-tota — DISMISSED** [review]

File: helm/values-dev.yaml [hunk 3]
```diff
@@ -45,10 +45,10 @@ microservice-chart:
   resources:
     requests:
       memory: "512Mi"
-      cpu: "200m"
+      cpu: "300m"
     limits:
       memory: "512Mi"
-      cpu: "200m"
+      cpu: "500m"
```
Comment (infantesimone) [inline helm/values-dev.yaml:51]: let's increase by only 100, let's try with 400

Comment (giovanniberti) [reply helm/values-dev.yaml]: Updated in 4185fd9

**Review by infantesimone — APPROVED** [review]

**Pull request merged**